typescript-test-samples/schema-and-contract-testing/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    semi: true,
+    trailingComma: "all",
+    singleQuote: true,
+    printWidth: 120,
+    tabWidth: 4
```

Comment:
`tabWidth` 4 here is causing a lot of `prettier` errors in the `*.ts` files, which are indented with 2 spaces.

Could you either:
1. Update this config to use two spaces (preferred)
2. Update code to match prettier config here

Reply:
I copied this from the sibling folders ('typescript-test-sample' and 'apigw-lambda-dynamodb') and it's using 4 spaces there. In my *.ts files there's 4 spaces for a tab as well.

Reply:
Ok. We can't rule out a mis-configuration on my side either I suppose. Will take a look in some other editors validate what I'm getting.